docs/resources/rego_policy_instance.md

```diff
@@ -71,167 +71,19 @@ All templates use parameters defined as JSON, below is a list of all the corresp
 
 -> You can also use the Cyral API `GET` `/v1/regopolicies/templates` to retrieve all existing templates and their corresponding parameters schema.
 
-### Data Firewall (data-firewall)
```

Comment:
Should we retain these templates in the doc but mark them as deprecated and note that no new instances of this type can be created?

Reply:
Yes, I can do that change, put the ones that are not deprecated on top and following a deprecation note list the others.